acceptance/bundle/variables/complex-simple/databricks.yml

```diff
@@ -0,0 +1,28 @@
+# This example works and properly merges resources.jobs.job1.job_clusters.new_cluster and ${var.cluster}.
+# retaining num_workers, spark_version and overriding node_type_id.
+# However, it prints a warning "expected map, found string" which is unnecessary.
+bundle:
+  name: TestResolveComplexVariable
+
+variables:
+  type: "complex"
```

Comment:
The warning is legit, this belongs one level deeper.

Reply:
Thanks! Fixed in https://github.com/databricks/cli/pull/2154/commits/402858403d7ea0184c32517231a802cd3854c856